articles/sso/v2/single-page-apps-sso.md

```diff
@@ -0,0 +1,134 @@
+---
+description: How to implement client-side SSO on single page applications.
+toc: true
+---
+
+# Client-Side SSO on Single Page Applications
+
+:::panel-warning Prerequisites
+This article assumes that you have already created an Auth0 Client (with its type set to **Single Page Web Applications**) with the **OIDC Conformant** flag enabled.
+:::
+
+To bypass displaying the Lock screen when logging in a user (a process known as [*silent authentication*](#silent-authentication)), you must:
+
+* Enable the **Use Auth0 instead of the IdP to do Single Sign On** flag in the [Auth0 Client's settings tab](${manage_url});
+* Have a SSO cookie for the tenant's domain (in other words, the user has previously signed in and their saved cookie is still valid);
+* Pass the name of the user's Connection to Auth0 for authentication. You can do this by:
+  * Including it as a parameter when calling the `signin` function of the [auth0.js library](/libraries/auth0js);
+  * Passing the `connection` query string parameter when calling the [Authentication API's `/authorize` endpoint](/api/authentication#login).
```

Comment:
Update API URL to `/api/authentication##implicit-grant`. It's the same endpoint, but the one I'm referring to is specific to Implicit Grant.